src/CabanaPD_Particles.hpp

```diff
@@ -428,6 +429,8 @@ class Particles<MemorySpace, PMB, Dimension>
                  [[maybe_unused]] const double output_time,
                  [[maybe_unused]] const bool use_reference = true )
     {
+        _timer.start();
```

Comment:
@streeve We do not have before "Timer _timer;"

Reply:
At the bottom of this class